scripts/contributor_dashboard_debug.py

```diff
@@ -0,0 +1,262 @@
+# coding: utf-8
+#
+# Copyright 2022 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This script assists development of the contributor dashboard. When the
+"contributor dashboard debug" flag is enabled, the start script will call the
+"populate_debug_data" function, which does the following:
+    1. Create an admin user with the username "a" and assign curriculum,
+    translation, and question admin rights to "a".
+    2. Create a non-admin user with the username "b" and give the user "submit
+    question" rights.
+    3. Populate sample lessons for translations. This is achieved by reproducing
+    the "Load dummy new structures data" admin action.
+    4. Add the topics linked to the lessons above to a classroom. This is needed
+    for the topics, if published, to show up in the topic selectors and for the
+    linked opportunities to show up in the "Submit Question" tab.
+"""
+
+from __future__ import annotations
+
+import contextlib
+import hashlib
+import json
+import os
+
+from core import feconf
+from core.constants import constants
+
+import firebase_admin
+from firebase_admin import auth as firebase_auth
+
+import requests
+from typing import Any, Dict, Iterator, List
+
+FIREBASE_AUTH_EMULATOR_HOST = 'localhost:%s' % feconf.FIREBASE_EMULATOR_PORT
```

Comment:
```suggestion
FIREBASE_AUTH_EMULATOR_HOST: Final = 'localhost:%s' % feconf.FIREBASE_EMULATOR_PORT
```

Reply:
I think we can make all the constants `Final`.

Ditto elsewhere

Reply:
Thanks. I added `Final` for all constants.